vuepress/docs/next/docs/reference/bundle-upgrade.md

```diff
@@ -0,0 +1,137 @@
+
+### 3. Add Micro Frontends & Microservices
+ 
+1. From inside your bundle project folder, add your micro frontend to the bundle descriptor `entando.json`:
+``` sh
+ent bundle mfe add YOUR-MFE1
+```
+>Docker-based bundles have a centralized bundle descriptor that includes specifications for bundle entities such as micro frontends, microservices, APIs, DBMS, and ID management. 
+
+2. Copy the MFE files into the `YOUR-BUNDLE-NAME/microfrontends` directory. For instance, for an MFE named YOUR-MFE1: 
+
+``` sh
+mv YOUR-ORIGIN-DIRECTORY/YOUR-MFE1/{.,}* microfrontends/YOUR-MFE1
+```
+>Micro frontends and microservices are partitioned into their own directories in v5 bundles to streamline the CI/CD process.
+
+3. Add your microservice:
+``` sh
+ent bundle ms add YOUR-MS1
+```
+4. Copy YOUR-MS1 files to the new location:
+``` sh
+mv YOUR-ORIGIN-DIRECTORY/YOUR-MS1/{.,}* microservicess/YOUR-MS1
+```
+5. Repeat steps 1 through 4 for all micro frontends and microservices.
+
+### 4. Add API Claims
+1. Add an API claim to connect `YOUR-MFE1` to a microsersive as needed. The connection information is added to the descriptor `entando.json`.
+```shell
+ent bundle api add YOUR-MFE1 YOUR-API --serviceName=YOUR-MS1 --serviceUrl=http://localhost:8081
+```
+
+2. Repeat the previous step for all APIs as needed.
+
+### 5. Add Custom Commands to Test Locally
+
+For local development and testing, a custom command is needed in the `entando.json` to use a different port for each MFE as shown below: 
+``` 
+    "microfrontends": [
+        {
+            "name": "YOUR-MFE1",
+            "customElement": "YOUR-MFE1",
+            "stack": "react",
+            "type": "widget",
+            "group": "free",
+            "publicFolder": "public",
+            "commands": {
+                "run": "npm install && PORT=3000 npm start"
+            },
+            "apiClaims": [
+                {
+                    "name": "YOUR-API1",
+                    "type": "internal",
+                    "serviceName": "YOUR-MS1"
+                }
+            ]
+        },
+        {
+            "name": "YOUR-MFE2",
+            "customElement": "YOUR-MFE2",
+            "stack": "react",
+            "type": "widget",
+            "group": "free",
+            "publicFolder": "public",
+            "titles": {
+                "en": "YOUR-MFE2",
+                "it": "YOUR-MFE2"
+            },
+            "commands": {
+                "run": "npm install && PORT=3001 npm start"
+            },
+            "apiClaims": [
+                {
+                    "name": "YOUR-API2",
+                    "type": "internal",
+                    "serviceName": "YOUR-MS2"
+                }
+            ]
+        }
+    ]
+```
+>In production, all microservices must run on port 8081.
+
+### 6. Add Auxiliary Services
+Copy auxiliary services such as DBMS or Keycloak into the `svc` directory as needed and enable them for local testing. For a Keycloak service:
```

Comment:
This is need to be confirmed, but I think svc should contain docker-compose files (and configuration if needed) or equivalent to run auxiliary services.
This is basically what is happening with JHipster-generated bundles when we move them to v5.
It could be different with some use cases I don't know yet. However, at this moment this part sounds not clear on what we should have here. I would add more details about that to explain to the reader what is he doing.